src/docs/product/sentry-basics/tracing/event-detail.mdx

```diff
@@ -68,19 +68,19 @@ The span view may be limited to one project at a time if you're on the [Team pla
 
 </Alert>
 
-#### Traversing and Nesting Transactions
```

Comment:
we might want to be explicit about "Nesting"

Reply:
What about "Traversing and Adding Transactions" as the heading? Does that address your concern @dashed or do you think we need to stick with the "nesting" terminology?

Reply:
I think we should be explicit that "Traversing" isn't the same as embedding/nesting/inlining/adding a transaction. I don't have a strong opinion of what the new product feature should be called.

Reply:
I've added my heading suggestion on line 71